scripts/lint-python

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+RC=0
+
+pip install -q -e ${SCRIPT_DIR}/../python/ || exit 1
```

Comment:
Avoid doing installation of packages in this script.  This should assume that its dependencies are already installed.